packages/bierzo-wallet/src/utils/test/persona.ts

```diff
@@ -21,9 +21,13 @@ export const submitExtensionCreateWalletForm = async (page: Page, password: stri
   await page.type(`input[name="${PASSWORD_CONFIRM_FIELD}"]`, password);
   await page.click(`input[name="${TERMS_ACCEPT_FIELD}"]`);
   await page.click('button[type="submit"]');
-  await sleep(1000);
+  await sleep(1500); // TODO: find direct way to detect if page was change to next step
```

Comment:
We can use `waitFor` puppeteer api method (https://github.com/GoogleChrome/puppeteer/blob/v1.20.0/docs/api.md#pagewaitforselectororfunctionortimeout-options-args). We can wait for next step id in this case this is `SECOND_STEP_CREATE_WALLET_ROUTE` from packages/sanes-browser-extension/src/routes/create-wallet/components/ShowPhraseForm.tsx

Reply:
Good idea.

So we would do

```ts
await page.waitForSelector(`#${SECOND_STEP_CREATE_WALLET_ROUTE}`);
```

? I was already thinking in the direction but only with routes (https://github.com/iov-one/ponferrada/issues/729). But it would be nice if the routes are not necessary to fix this.

Reply:
Yes, you're right, this is exactly what I had in mind. Routes is not required in this case, also there is no route changes between wallet creation steps.

Reply:
Great, much better than the sleeps as it will save much testing time on average and is more stable in slow situations

Reply:
And in this case #729 is not required any more.

Reply:
I implemented the suggested approach now. Much better! Main commit for the fix is https://github.com/iov-one/ponferrada/pull/728/commits/7975d6971841c0027869007aa7312607a1b1ea5b